src/main/java/com/hbm/items/tool/ItemPipette.java

```diff
@@ -0,0 +1,175 @@
+package com.hbm.items.tool;
+
+import api.hbm.fluid.IFillableItem;
+
+import com.hbm.inventory.fluid.FluidType;
+import com.hbm.inventory.fluid.Fluids;
+import com.hbm.items.ModItems;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.IIcon;
+import net.minecraft.world.World;
+
+import java.util.List;
+
+public class ItemPipette extends Item implements IFillableItem {
+
+    public ItemPipette() {
+        this.setMaxDamage(0);
+    }
+
+    @SideOnly(Side.CLIENT) protected IIcon overlayIcon;
+
+    public int amount = this.getMaxFill();
```

Comment:
there's only one item instance and therefore that value is shared for each pipette, you have to use NBT for that